scripts/ci-e2e.sh

```diff
@@ -29,8 +29,8 @@ source "${REPO_ROOT}/scripts/environment.sh"
 # Clone dev-env repo
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=main
+M3_DEV_ENV_REPO="https://github.com/Nordix/metal3-dev-env.git"
+M3_DEV_ENV_BRANCH=bump/capi-v1.7.0beta 
```

Comment:
shouldnt we merge that to main first it looks weird to merge this to main than revert it

Reply:
Check the commit message

Reply:
Removed now.